src/plugins/Trace.ts

```diff
@@ -202,6 +228,119 @@ export class Trace {
         return originalEvent;
     }
 
+    private enrichAuditInfo(auditInfo: AuditInfo) {
```

Comment:
I think, these logics should be handled in the integration itself. Otherwise, with every new integration this part will be getting worse

Reply:
You already have `domainName` and `className` as argument to `ThundraSpan`. You can pass them while creating span and then map them as 
- `domainName` of span to `contextGroup` of audit info
- `className` of span to `contextType` of audit info
- `operationName` of span to `contextName` of audit info
- `tags` of span to `props` of audit info
- There might be mismatch between tags and props (such as `operation.type` and `operationType`) with new data model. Therefore, for now you can put both versions of tags for now with a `TODO` comment which says this will be removed with new data model compatible agent version